c10/core/Scalar.h

```diff
@@ -176,7 +185,7 @@ class C10_API Scalar {
   // We can't set v in the initializer list using the
   // syntax v{ .member = ... } because it doesn't work on MSVC
 
-  enum class Tag { HAS_d, HAS_i, HAS_z, HAS_b };
+  enum class Tag { HAS_d, HAS_i, HAS_z, HAS_b, HAS_n };
```

Comment:
`HAS_n` is a weird name for saying it doesn't have anything :P Worth a comment, or Note reference, here.